src/interface/src/app/plan/create-scenarios/create-scenarios.component.html

```diff
@@ -1,4 +1,10 @@
-<div class="create-scenarios-panel mat-elevation-z2">
+<app-resource-unavailable
+  resource="scenario"
+  *ngIf="scenarioNotFound"></app-resource-unavailable>
+
+<div
+  class="create-scenarios-panel mat-elevation-z2"
+  [ngClass]="{ hidden: scenarioNotFound }">
```

Comment:
here I cannot use an ngIf because it breaks all the form initiation/creation, so just hiding it 🤷